db/migrate/20230714030147_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+    def change
+      create_table :posts do |t|
+        t.string :title
+        t.text :text
+        t.integer :comments_counter
+        t.integer :likes_counter
+        t.references :author, null: false, foreign_key: true
+  
```

Comment:
- [ ] Ouch! this is a close one but the code you have at line `8` is trying to reference a table called `authors` when creating the association (default implementation) which doesn't exist instead we want it to refer to the `users` table so we should explicitly mention the table name as shown below
```rb
t.references :author, foreign_key: { to_table: 'users' }
```